app/components/dashboard/run/left-panel/component.js

```diff
@@ -165,5 +166,23 @@ export default Component.extend(FullScreenMixin, {
         denyDelete() {
             return true;
         },
+        
+        exportZip(id) {
+          const token = this.get('tokenHandler').getWholeTaleAuthToken();
+          let url = `${config.apiUrl}/tale/${id}/export`;
+          window.open(url + '?token=' + token + '&taleFormat=native', '_newtab');
+        },
+
+        exportBag(id) {
+          const token = this.get('tokenHandler').getWholeTaleAuthToken();
+          let url = `${config.apiUrl}/tale/${id}/export`;
+          window.open(url + '?token=' + token + '&taleFormat=bagit', '_newtab');
```

Comment:
Why would we want to open a tab if it's immediately closed? It just causes my web browser to flicker. I believe my original suggestion didn't exhibit this behavior.